### DISCUSSION TITLE
How to add keymap/shortcut to manually import a PHP class?

### DISCUSSION THREAD

**Discussion by mansoorkhan96** [zed-industries/zed#6670] [Feature Requests]
In vscode i have a extension that lets me set a keybinding to manually import a class 

```json
{
    "key": "cmd+k cmd+i",
    "command": "namespaceResolver.import"
},
```

**Comment by samuel-fonseca** [comment]
@mansoorkhan96 did you ever find an answer for this?

**Reply by mansoorkhan96** [reply]
> @mansoorkhan96 did you ever find an answer for this?

Unfortunately, no. I have not tried recently though. It might be possible now that zed is getting more stable and feature rich.

**Reply by samuel-fonseca** [reply]
@mansoorkhan96 looking at what's available it doesn't appear the case. Will maybe have to look at implementing my own, can't find any open source options...

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
